### ISSUE TITLE
Creative tab ordering should be changed

### ISSUE THREAD

**Issue by CriticalPickle** [CriticalPickle/critical-crates#26]
Creative tab ordering should be changed for easier access to tools and upgrades instead of having to scroll through all crate types. This could easily become a larger problem as more crates, upgrades, or other features are added.

**Comment by CriticalPickle** [comment]
Complete for both 1.21.1 and 1.21.11!
